_includes/head.html

```diff
@@ -1,11 +1,21 @@
 <head>
+
+    <title>{% if page.title %}{{ page.title }}{% else %}{{ site.title }}{% endif %}</title>
+    <link rel="icon" type="image/png" href="./img/logo/logoOA_svg.png">
```

Comment:
Is it not a good idea to add a scaled version of the logo for the favicon?

Reply:
I think the favicon scales down nicely in the title bar from large dimensions too. So I think this will be  fine @Cadair